package.json

```diff
@@ -59,7 +59,8 @@
     "test": "npm run lint && npm run test:server && npm run test:client",
     "test:client": "karma start --single-run",
     "test:server": "mocha --timeout 20000 --compilers js:babel-core/register test/bootstrap.test.js test/controller/**/*.js",
-    "test:watch": "concurrently \"mocha -w --compilers js:babel-core/register\" \"karma start\""
+    "test:watch": "concurrently \"mocha -w --compilers js:babel-core/register\" \"karma start\"",
+    "version": "node -e 'console.log(require(\"./package\").version)'"
```

Comment:
I don't really think this is a problem, but I agree that it's not a great solution. So I guess I'm fine with what you had initially.